src/main/java/no/dcat/datastore/domain/dcat/vocabulary/AT.java

```diff
@@ -0,0 +1,25 @@
+package no.dcat.datastore.domain.dcat.vocabulary;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+
+public class AT {
+    private static final Model m = ModelFactory.createDefaultModel();
+    public static final String uri = "http://publications.europa.eu/ontology/authority/";
+    public static final Resource NAMESPACE;
+    public static final Property authorityCode;
+
+    public AT() {
```

Comment:
Nå merker jeg at jeg ikke jobber så mye direkte med java, men kan du ikke bare droppe denne?

Reply:
Trenger Propertien for å hente den ut fra grafen til Jena